libs/blocks/global-navigation/global-navigation.js

```diff
@@ -123,26 +122,26 @@ class Gnav {
     this.ready = this.ready || new Promise(async (resolve) => {
       this.el.removeEventListener('click', this.loadDelayed);
       const [
-        { MenuControls },
         decorateDropdown,
         { appLauncher },
         ProfileDropdown,
         Search,
+        KeyboardNavigation,
       ] = await Promise.all([
-        loadBlock('./utilities/delayed-utilities.js'),
         loadBlock('./blocks/navDropdown/dropdown.js'),
         loadBlock('./blocks/appLauncher/appLauncher.js'),
         loadBlock('./blocks/profile/dropdown.js'),
         loadBlock('./blocks/search/gnav-search.js'),
+        loadBlock('./utilities/keyboard/index.js'),
```

Comment:
Do we need this to be loaded as soon as possible? I was thinking maybe we can delay until needed to avoid potential performance issues.